backend/src/models/mural.model.ts

```diff
@@ -79,6 +81,14 @@ Mural.init(
       type: DataTypes.INTEGER,
       references: { model: "boroughs", key: "id" },
     },
+    coordinates: {
+      type: DataTypes.GEOMETRY("POINT", 4326),
```

Comment:
Would it assign a default SRID without 4326?

Reply:
lol honestly the concept of a SRID went over my head. several examples used 4326 tho. https://gis.stackexchange.com/questions/131363/choosing-srid-and-what-is-its-meaning